helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -57,6 +57,9 @@ receivers:
     extra_metadata_labels:
       - container.id
 
+  smartagent/signalfx-forwarder:
+    type: signalfx-forwarder
+
```

Comment:
Should `listenAddress` also be configured to match the default agent config (`0.0.0.0:9080`)?

Reply:
I wasn't sure tbh since this isn't in the SA config map, but if that's the default everything else uses makes sense.